index.d.ts

```diff
@@ -165,7 +176,7 @@ export type PinResponse = {
   reference?: string
 }
 
-export type StatusOptions = {
+export interface StatusOptions extends RequestOptions {
```

Comment:
What is the benefit of using `interface` and `extends` over `type` and `&`?

Reply:
> What is the benefit of using `interface` and `extends` over `type` and `&`?

I'm not convinced there is one, yet @hugomrdias trained me to prefer interfaces over types with an arguments articulated in this post https://ncjamieson.com/prefer-interfaces/